tests/test_graph_types.py

```diff
@@ -1,11 +1,80 @@
 import inspect
 import json
+import sys
+import textwrap
+
+import pytest
 
 from hamilton import graph_types, node
 
 from tests import nodes as test_nodes
 
 
+@pytest.fixture()
+def func_a():
+    """Default function implementation"""
+
+    return textwrap.dedent(
+        """
+    def A(external_input: int) -> int:
+        return external_input % 7
+    """
+    )
+
+
+@pytest.fixture()
+def func_a_body():
+    """The function A() has modulo 5 instead of 7"""
+
+    return textwrap.dedent(
+        """
+    def A(external_input: int) -> int:
+        return external_input % 5
+    """
+    )
+
+
+@pytest.fixture()
+def func_a_docstring():
+    """The function A() has a docstring"""
+
+    return textwrap.dedent(
+        '''
```

Comment:
nit -- quote styles should match the above

Reply:
Ohh, it's because I have the docstring within the triple-quoted string. I used `'''` for the docstring instead.

I was wondering why